.github/workflows/linting-yaml.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint YAML files
+
+on:
+  push:
+    paths:
+      - "products/**"
+      - .github/workflows/linting-yaml.yml
+  pull_request:
+    branches: [master]
+    paths:
+      - "products/**"
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Check file naming
+        run: |
+          files=$(find products/*.yml | wc -l)
+          if [ $files -gt 0 ]; then echo "File found with .yml extension, please rename with .yaml" && exit 1; fi
+
+      - name: yaml-lint
+        run:
+          yamllint products/*.yaml
```

Comment:
should do `cd products; yamllint .`

Reply:
it  picks up an error for archived folder, maybe we need a README in archived folder,

Reply:
Why?